modules/backend/classes/FormField.php

```diff
@@ -449,11 +461,27 @@ protected function filterAttributes($attributes, $position = 'field')
         if ($position == 'field' && $this->readOnly) {
             $attributes = $attributes + ['readonly' => 'readonly'];
 
+            // https://stackoverflow.com/questions/155291/can-html-checkboxes-be-set-to-readonly
             if ($this->type == 'checkbox' || $this->type == 'switch') {
                 $attributes = $attributes + ['onclick' => 'return false;'];
             }
         }
 
+        if ($position == 'field' && $this->mask != '' && $this->type == 'text') {
```

Comment:
Don't use `==` please, that's just asking for potential headaches

Reply:
Most probably @LukeTowers was saying to change == -> === :) 'empty' check also good one 👍

Reply:
Thank you for answer. I modified the code.

Reply:
@w20k is correct @gergo85; thanks to both of you.